src/graph/optimizer/rule/PushFilterDownHashInnerJoinRule.cpp

```diff
@@ -82,7 +82,7 @@ StatusOr<OptRule::TransformResult> PushFilterDownHashInnerJoinRule::transform(
     newGroupNode->dependsOn(const_cast<OptGroup*>(newJoinGroup));
   } else {
     newInnerJoinNode->setOutputVar(oldFilterNode->outputVar());
-    newInnerJoinNode->setColNames(oldFilterNode->colNames());
+    newInnerJoinNode->setColNames(oldInnerJoinNode->colNames());
```

Comment:
Why the output column names of oldFilterNode are different from oldInnerJoinNode?

Reply:
yi ge shi filter, yi ge shi join, ying gai bu yi yang ba.